test/mocha/string-literal.js

```diff
@@ -81,16 +81,16 @@ describe("String literals", function() {
 
     it("Should not unescape unpaired surrogates", function() {
         var code = [];
-        for (var i = 0; i <= 0xF; i++) {
+        for (var i = 0; i <= 0xF; i+=10) {
```

Comment:
@fabiosantoscode Please back out all changes to `test/mocha/string-literal.js`.

This is an extremely important test for minification integrity and has caught many bugs.